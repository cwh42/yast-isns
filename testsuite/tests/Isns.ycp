{

// testedfiles: Isns.ycp

include "testsuite.ycp";
TESTSUITE_INIT([], nil);

import "Isns";

DUMP("Isns::Modified");
TEST(``(Isns::Modified()), [], nil);

}
