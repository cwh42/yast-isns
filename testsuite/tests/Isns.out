Dump	Isns::Modified
Return	false
